spec/interactors/send_timer_reminders_spec.rb

```diff
@@ -19,7 +19,37 @@
       pto: false
     })
 
-    SendTimerReminders.call
+    SendTimerReminders.call(time_zone: user_1.time_zone)
+
+    expect(mailbox_for(user_1.email)).to be_empty
+    expect(day_1.reload.timer_reminder_sent).to be_falsey
+
+    open_last_email_for(user_2.email)
+    expect(current_email).to have_i18n_subject_for("timer_reminder")
+    expect(day_2.reload.timer_reminder_sent).to be_truthy
+  end
+
+  it "only reminds users who have passed the time threshold in their time zone" do
+    user_1 = create(:user, time_zone: "Cairo")
+    user_2 = create(:user, time_zone: Figaro.env.harvest_time_zone)
```

Comment:
Let's hard code `user_2`'s time zone here. Otherwise, it's possible that this test would fail for certain values of the Harvest time zone configuration.